cohort-engine/src/test/resources/cql/includes/Breast-Cancer-Screening.cql

```diff
@@ -19,14 +19,7 @@ define "MeetsInclusionCriteria":
   and "Female"
 
 define "MeetsExclusionCriteria":
-  "Breast Cancer"
-  or "Bilateral Mastectomy"
-  or "Unilateral mastectomy left and right."
-  or "Recent Mammogram"
-
-define "Unilateral mastectomy left and right.":
-  "Unilateral Mastectomy Left"
-  and "Unilateral Mastectomy Right"
+  false
```

Comment:
Did you mean to change this? I see the dummy CQLs so wasn't sure.

Reply:
Intentional. This was failing to compile and I never noticed it. I recently added a check for translation errors and this popped out, so I had to fix it.